make-readme-markdown.el

```diff
@@ -384,6 +385,21 @@ any license found."
   (print-license-badge lines)
   (print-status-badges lines))
 
+(defun print-emacs-icon (lines)
+  "Print emacs icon to generate a fancy README.md."
+  (let* ((package-url (plist-get file-headers 'URL))
+	 (logo-url "<img src=\"https://www.gnu.org/software/emacs/images/emacs.png\" alt=\"Emacs Logo\" width=\"80\" height=\"80\" align=\"right\">"))
+    (if package-url
+	(progn
+	  (message "Adding emacs icon with URL: %s" package-url)
+	  (princ (format "<a href=\"%s\">%s</a>\n"
+			 package-url
+			 logo-url)))
+	(progn
```

Comment:
this is overindented (highlight the whole function and do ``)

Also, the `else` clause doesn't need a `progn`.

All told, this function should look more like this:

```elisp
(defun print-emacs-icon ()
  "Print emacs icon to generate a fancy README.md."
  (let* ((package-url (plist-get file-headers 'URL))
	 (logo-url "<img src=\"https://www.gnu.org/software/emacs/images/emacs.png\" alt=\"Emacs Logo\" width=\"80\" height=\"80\" align=\"right\">"))
    (if package-url
	(progn
	  (message "Adding emacs icon with URL: %s" package-url)
	  (princ (format "<a href=\"%s\">%s</a>\n"
			 package-url
			 logo-url)))
      (message "Adding emacs icon without URL")
      (princ (format "%s\n"
                     logo-url)))))
```

Reply:
woops, never finished typing my comment about indenting.  Highlight the whole function and do `indent-region` (bound to `C-M-\` by default).

Reply:
Hi @mgalgs :

I update my PR by force push, could you help me review again?

Fixed:
1. code intent by `M-x indent-region`
2. remove `progn` on else block

Reply:
Awesome!! Can you also remove `lines` from your new `print-emacs-icon` function? That variable appears to be unused...

Reply:
done :)